diff --git a/test/seed_database.sql b/test/seed_database.sql
index fa9ad39..fe6a22d 100644
--- a/test/seed_database.sql
+++ b/test/seed_database.sql
@@ -4,7 +4,7 @@ VALUES ('account1', 'user1', '2024-04-17 09:35:50.651027026+00:00', 'netbird.sel
 
 -- Seed Users table
 INSERT INTO users (id, account_id, role, is_service_user, non_deletable, blocked, created_at, issued)
-VALUES ('user1', 'account1', 'owner', 0, 0, 0, '2024-08-12 00:00:00', 'api');
+VALUES ('user1', 'account1', 'owner', 1, 0, 0, '2024-08-12 00:00:00', 'api');
 
 -- Seed Groups table
 INSERT INTO groups (id, account_id, name, issued, integration_ref_id, integration_ref_integration_type)
