diff --git a/test/management.json b/test/management.json
index 2843d96..e8f906c 100644
--- a/test/management.json
+++ b/test/management.json
@@ -15,7 +15,7 @@
     "Password": ""
   },
   "Datadir": "/var/lib/netbird",
-  "DataStoreEncryptionKey": "Qb/a9fEuGOUQQqu3EZW80btl9ig5csuDUfa31RIJ/8M=",
+  "DataStoreEncryptionKey": "Zm4cgxLh3jzUvul22IyVvyLj3ldLJMcV/jdpXLnB6eQ=",
   "EmbeddedIdP": {
     "Enabled": true,
     "Issuer": "http://localhost:8080/oauth2"
diff --git a/test/seed_database.sql b/test/seed_database.sql
index fa9ad39..fe6a22d 100644
--- a/test/seed_database.sql
+++ b/test/seed_database.sql
@@ -4,7 +4,7 @@ VALUES ('account1', 'user1', '2024-04-17 09:35:50.651027026+00:00', 'netbird.sel
 
 -- Seed Users table
 INSERT INTO users (id, account_id, role, is_service_user, non_deletable, blocked, created_at, issued)
-VALUES ('user1', 'account1', 'owner', 0, 0, 0, '2024-08-12 00:00:00', 'api');
+VALUES ('user1', 'account1', 'owner', 1, 0, 0, '2024-08-12 00:00:00', 'api');
 
 -- Seed Groups table
 INSERT INTO groups (id, account_id, name, issued, integration_ref_id, integration_ref_integration_type)
